3i-deploy/Gemfile

```diff
@@ -16,3 +16,5 @@ gem 'berkshelf'
 
 gem "test-kitchen"
 gem "kitchen-vagrant"
+gem 'multipart-post'
+gem 'slackr', :git => 'git://github.com/risk-io/slackr.git'
```

Comment:
If I had to guess, I'd say the issue might be here. There's a couple things that are done differently here than the other dependencies listed in this file:

- **Single quotes rather than double.** - I really have no idea if this makes any difference to whatever version of Ruby is running on Opsworks, but it's different than the other entries).
- **The git repo cited inline.** - I think this is the more likely culprit.

Reply:
Whoops, I missed your comments from awhile ago - thanks!

Hmmm, I'll play around with it, see if that happens to fix it.  If it is that simple of an issue, 🤦‍♂️